spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -165,12 +190,6 @@ public Sender stackdriverSender(GcpTraceProperties traceProperties,
 				.build();
 	}
 
-	@Bean
-	@ConditionalOnMissingBean
-	public BytesEncoder<Span> spanBytesEncoder() {
```

Comment:
What dose make you to remove this bean altogether?

Reply:
That bean was needed to create a `Reporter` (because it was heavily depending on sleuth-zipkin): https://github.com/spring-cloud/spring-cloud-sleuth/blob/master/spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinAutoConfiguration.java#L77

In the new configuration there is a dedicated reporter for Stackdriver so we don't need to have this bean anymore.
What was previously `.metrics(reporterMetrics).build(spanBytesEncoder);` (sleuth-zipkin) is now `.metrics(reporterMetrics).build(StackdriverEncoder.V1);` (gcp-trace)